opensearch/src/main/java/org/opensearch/sql/opensearch/response/OpenSearchResponse.java

```diff
@@ -91,7 +94,21 @@ public Iterator<ExprValue> iterator() {
       }).iterator();
     } else {
       return Arrays.stream(hits.getHits())
-          .map(hit -> (ExprValue) exprValueFactory.construct(hit.getSourceAsString())).iterator();
+          .map(hit -> {
+            ExprValue docData = exprValueFactory.construct(hit.getSourceAsString());
+            if (hit.getHighlightFields().isEmpty()) {
+              return docData;
+            } else {
+              ImmutableMap.Builder<String, ExprValue> builder = new ImmutableMap.Builder<>();
+              builder.putAll(docData.tupleValue());
+              var hlBuilder = ImmutableMap.<String, ExprValue>builder();
```

Comment:
I personally don't like `var` (but I realize I'm in the minority on this team) but you should be consistent.  The builder above has an explicit type, and you use var here.

Reply:
Also, hlBuilder is not a great name for variable.  We don't save any money by typing fewer characters, and we shouldn't make the readers need to guess what the 'hl' stands for.  
This isn't a highlight specific class, and 'hl' isn't a known abbreviation. 
I'd recommend `highlightMapBuilder`.